### DISCUSSION TITLE
is there any way to set a cross-seed category?

### DISCUSSION THREAD

**Discussion by ScaredyCatX** [StuffAnThings/qbit_manage#401] [Q&A]
is there any way to set the category based on a tag?  if I cross-seed something from tracker A to tracker B. The category of the cross-seeded file is set as tracker A when I prefer if it was set to the tracker it's being cross-seeded to, which is tracker B.
basically, what i want is something like this: if tag contains "cross-seed" set the category to {whatever tracker in the tags}.

**Comment by bakerboy448** [comment]
Category-by-tracker exists and takes priority over path-based category — set `cat` under the `tracker:` section:
```yaml
tracker:
  trackerB.example.com:
    tag: trackerB
    cat: TrackerB-Category
```
This resolves before the save-path-based `cat:` fallback (`modules/core/category.py`), which is likely why cross-seeds currently inherit the original's category — save-path matching has no tracker awareness and cross-seeds typically share a save path.

One caveat that matters for "destination" tracker specifically: matching isn't destination-aware. It loops all tracker URLs attached to that torrent entry against your configured `tracker:` keywords **in the order you declared them**, and returns on the **first** keyword that matches **any** attached URL (`modules/qbittorrent.py`). If a single torrent entry ends up with both source and destination tracker URLs attached, and the source tracker's keyword is declared first in your config, the source wins even if you want the destination's category. Order your `tracker:` entries with the desired destination keyword first, or use sufficiently unique keywords, to control this.

If your cross-seed tool instead creates a separate torrent entry per tracker (only one tracker URL each), this works cleanly with no ordering concern.

Also worth checking: if the category's save path differs from where the cross-seed data lives and Automatic Torrent Management is on, qBittorrent may move the data — which can break the shared-data relationship cross-seeding depends on.

`cat_update_all` defaults to `true`, so already-categorized cross-seeds get corrected automatically on the next run once you add the tracker `cat` — no extra setting needed.
